[FR] Add `Ranking_Criteria\Difficulty_Naming`, `Ranking_Criteria_Council`, `Scaling_BPM`, `Timing_Songs_With_8-Signatures` (#6539)
ppy/osu-wiki: wiki/Ranking_Criteria/Timing_Songs_With_8-Signatures/fr.md

# Timing des chansons avec des signatures en #/8

De temps en temps, des chansons sont mappées avec une signature temporelle qui n'est pas divisée en quarts de note (par exemple `3/4`, `4/4`, `7/4`, ...) mais en croches (par exemple `6/8`, `7/8`, ...). Comme l'éditeur actuel ne supporte pas ces signatures #/8 et que nous ne savons pas si cela va changer dans un futur proche, un ajout a été fait à la règle concernant cette question sur la [section correspondante des critères de classement](/wiki/Ranking_Criteria#timing).

> Afin de rendre ce tableau plus compréhensible et d'indiquer plus précisément ce que l'on entend par là, ce guide va maintenant expliquer un peu de théorie musicale pertinente derrière tout cela, et donner des exemples de la façon dont cela se traduit en osu! Pour cette raison, [troisbeatmaps] (https://assets.ppy.sh/media/wiki/TimeSignatures.rar) ont été préparés pour que vous puissiez les télécharger, et ils seront utilisés comme référence dans ce guide.

Author: ```suggestion
Afin de rendre ce tableau plus compréhensible et d'indiquer plus précisément ce que l'on entend par là, ce guide va maintenant expliquer un peu de théorie musicale pertinente derrière tout cela, et donner des exemples de la façon dont cela se traduit en osu! Pour cette raison, [trois beatmaps](https://assets.ppy.sh/media/wiki/TimeSignatures.rar) ont été préparés pour que vous puissiez les télécharger, et ils seront utilisés comme référence dans ce guide.
```

## Théorie de la musique

> En général, une mesure contient un certain nombre de temps. En notation musicale, un temps est généralement égal à une noire ♩, et une noire ♩ est divisée en deux croches ♪♪. Par conséquent, le terme "battements par minute" (BPM) décrit la tempo d'une chanson en indiquant combien de noires ♩ sont jouées en une minute. Dans osu!, les battements sont représentés par les ticks blancs dans la timeline, et les croches ♪ sont généralement représentées par des ticks rouges. Le début de chaque mesure est indiqué par un gros tick blanc.

Reviewer: ```suggestion
En général, une mesure contient un certain nombre de temps. En notation musicale, un temps est généralement égal à un quart de note ♩, et un quart de note ♩ est divisé en deux croches ♪♪. Par conséquent, le terme "battements par minute" (BPM) décrit la tempo d'une chanson en indiquant combien de quarts de notes ♩ sont joués en une minute. Dans osu!, les battements sont représentés par les ticks blancs dans la timeline, et les croches ♪ sont généralement représentées par des ticks rouges. Le début de chaque mesure est indiqué par un gros tick blanc.
```

> En tout cas, c'est comme ça que ça marche pour les signatures en #/4. La différence fondamentale avec les `#/8-signatures` est qu'un temps ne contient pas toujours seulement deux croches ♪, il peut aussi contenir trois ♪ ! Un bon exemple est la différence entre une signature en 3/4 et une signature en 6/8 : Les deux contiennent six ♪, mais alors que `3/4` les divise en trois groupes de deux (♪♪+♪♪+♪♪, \[2+2+2\]) en accentuant la première note de chaque paire, `6/8` les divise en deux groupes de trois (♪♪♪+♪♪, \[3+3\]) ! Ces "groupes" sont aussi appelés "subdivisions". Les signatures temporelles ne contenant que des subdivisions de trois croches ♪ (comme dans `6/8`) sont dites "composées", les signatures temporelles contenant des subdivisions avec des quantités variables de croches ♪ sont dites "complexes" ou "irrégulières" (par exemple `7/8`, nous y reviendrons plus tard). Lorsqu'un certain modèle composé de différentes signatures temporelles se répète régulièrement, on parle de "mesure mixte".

Reviewer: ```suggestion
En tout cas, c'est comme ça que ça marche pour les signatures en #/4. La différence fondamentale avec les `#/8-signatures` est qu'un battement ne contient pas toujours seulement deux croches ♪, il peut aussi contenir trois ♪ ! Un bon exemple est la différence entre une signature en 3/4 et une signature en 6/8 : Les deux contiennent six ♪, mais alors que `3/4` les divise en trois groupes de deux (♪♪+♪♪+♪♪, \[2+2+2\]) en accentuant la première note de chaque paire, `6/8` les divise en deux groupes de trois (♪♪♪+♪♪, \[3+3\]) ! Ces "groupes" sont aussi appelés "subdivisions". Les signatures temporelles ne contenant que des subdivisions de trois croches ♪ (comme dans `6/8`) sont dites "composées", les signatures temporelles contenant des subdivisions avec des quantités variables de croches ♪ sont dites "complexes" ou "irrégulières" (par exemple `7/8`, nous y reviendrons plus tard). Lorsqu'un certain modèle composé de différentes signatures temporelles se répète régulièrement, on parle de "mesure mixte".
```

> En tout cas, c'est comme ça que ça marche pour les signatures en #/4. La différence fondamentale avec les `#/8-signatures` est qu'un battement ne contient pas toujours seulement deux croches ♪, il peut aussi contenir trois ♪ ! Un bon exemple est la différence entre une signature en 3/4 et une signature en 6/8 : Les deux contiennent six ♪, mais alors que `3/4` les divise en trois groupes de deux (♪♪+♪♪+♪♪, \[2+2+2\]) en accentuant la première note de chaque paire, `6/8` les divise en deux groupes de trois (♪♪♪+♪♪, \[3+3\]) ! Ces "groupes" sont aussi appelés "subdivisions". Les signatures temporelles ne contenant que des subdivisions de trois croches ♪ (comme dans `6/8`) sont dites "composées", les signatures temporelles contenant des subdivisions avec des quantités variables de croches ♪ sont dites "complexes" ou "irrégulières" (par exemple `7/8`, nous y reviendrons plus tard). Lorsqu'un certain modèle composé de différentes signatures temporelles se répète régulièrement, on parle de "mesure mixte".

Reviewer 2: ```suggestion
En tout cas, c'est comme ça que ça marche pour les signatures en #/4. La différence fondamentale avec les `#/8-signatures` est qu'un battement ne contient pas toujours seulement deux huitièmes de note ♪, il peut aussi contenir trois ♪ ! Un bon exemple est la différence entre une signature en 3/4 et une signature en 6/8 : Les deux contiennent six ♪, mais alors que `3/4` les divise en trois groupes de deux (♪♪+♪♪+♪♪, \[2+2+2\]) en accentuant la première note de chaque paire, `6/8` les divise en deux groupes de trois (♪♪♪+♪♪, \[3+3\]) ! Ces "groupes" sont aussi appelés "subdivisions". Les signatures temporelles ne contenant que des subdivisions de trois huitièmes de note ♪ (comme dans `6/8`) sont dites "composées", les signatures temporelles contenant des subdivisions avec des quantités variables de huitièmes de note ♪ sont dites "complexes" ou "irrégulières" (par exemple `7/8`, nous y reviendrons plus tard). Lorsqu'un certain modèle composé de différentes signatures temporelles se répète régulièrement, on parle de "mesure mixte".
```

Reviewer 2: je sais pas pourquoi git a bugué ici, mais globalement j'ai remplacé les "croches" par des "huitièmes de note", et "la tempo" par "le tempo"

![comparaison des subdivisions entre une mesure à 3/4 et une mesure à 6/8](/wiki/shared/timing/34th_vs_68ths.png)

> Maintenant pour vous donner un premier exemple de ceci dans osu!, regardez la difficulté `3/4` dans la beatmap fourni de "America" de Leonard Bernstein - Dans cette difficulté, nous avons tout chronométré en `3/4-signature`, ce qui signifie que chaque mesure est divisée en trois temps par des ticks blancs. Lorsque vous tournez le BeatSnapDivisor sur `1/2`, chaque temps est divisé en deux par les ticks rouges dans la timeline - Un tick rouge représentant une croche ♪. Cela fonctionnerait parfaitement bien pour une chanson en `3/4`, mais si vous écoutez attentivement, vous pouvez entendre quelque chose de spécial sur cette chanson : Alors que la tempo général reste (approximativement) le même, le regroupement rythmique des ♪ change presque à chaque mesure - La première mesure de la partie mappée à `00:03:662` est divisée en \[3+3\], car elle n'a qu'une seule accentuation de plus sur le tick rouge à `00:04:232`. La deuxième mesure, qui commence à `00:04:801` est regroupée en \[2+2+2\], car elle a deux accentuations supplémentaires sur les ticks blancs à `00:05:181` et `00:05:561`. En d'autres termes, toutes les mesures ont la même longueur (6♪s), mais la moitié d'entre elles n'ont que deux temps (-> `6/8-mesure`) tandis que l'autre moitié a trois temps (-> `3/4-mesure`), ce qui en fait une mesure mixte ! Il y a différentes façons de gérer cela dans osu! qui ont toutes leurs avantages et leurs inconvénients, que nous verrons dans le prochain segment.

Reviewer: ```suggestion
Maintenant pour vous donner un premier exemple de ceci dans osu!, regardez la difficulté `3/4` dans la beatmap fournie de la chanson "America" de Leonard Bernstein - Dans cette difficulté, nous avons tout chronométré en `3/4-signature`, ce qui signifie que chaque mesure est divisée en trois temps par des ticks blancs. Lorsque vous réglez le BeatSnapDivisor sur `1/2`, chaque temps est divisé en deux par les ticks rouges dans la timeline - Un tick rouge représentant une croche ♪. Cela fonctionnerait parfaitement bien pour une chanson en `3/4`, mais si vous écoutez attentivement, vous pouvez entendre quelque chose de spécial sur cette chanson : Alors que la tempo général reste (approximativement) le même, le regroupement rythmique des ♪ change presque à chaque mesure - La première mesure de la partie mappée à `00:03:662` est divisée en \[3+3\], car elle n'a qu'une seule accentuation de plus sur le tick rouge à `00:04:232`. La deuxième mesure, qui commence à `00:04:801` est regroupée en \[2+2+2\], car elle a deux accentuations supplémentaires sur les ticks blancs à `00:05:181` et `00:05:561`. En d'autres termes, toutes les mesures ont la même longueur (6♪s), mais la moitié d'entre elles n'ont que deux temps (-> `6/8-mesure`) tandis que l'autre moitié a trois temps (-> `3/4-mesure`), ce qui en fait une mesure mixte ! Il y a différentes façons de gérer cela dans osu! qui ont toutes leurs avantages et leurs inconvénients, que nous verrons dans le prochain segment.
```

> Maintenant pour vous donner un premier exemple de ceci dans osu!, regardez la difficulté `3/4` dans la beatmap fournie de la chanson "America" de Leonard Bernstein - Dans cette difficulté, nous avons tout chronométré en `3/4-signature`, ce qui signifie que chaque mesure est divisée en trois temps par des ticks blancs. Lorsque vous réglez le BeatSnapDivisor sur `1/2`, chaque temps est divisé en deux par les ticks rouges dans la timeline - Un tick rouge représentant une croche ♪. Cela fonctionnerait parfaitement bien pour une chanson en `3/4`, mais si vous écoutez attentivement, vous pouvez entendre quelque chose de spécial sur cette chanson : Alors que la tempo général reste (approximativement) le même, le regroupement rythmique des ♪ change presque à chaque mesure - La première mesure de la partie mappée à `00:03:662` est divisée en \[3+3\], car elle n'a qu'une seule accentuation de plus sur le tick rouge à `00:04:232`. La deuxième mesure, qui commence à `00:04:801` est regroupée en \[2+2+2\], car elle a deux accentuations supplémentaires sur les ticks blancs à `00:05:181` et `00:05:561`. En d'autres termes, toutes les mesures ont la même longueur (6♪s), mais la moitié d'entre elles n'ont que deux temps (-> `6/8-mesure`) tandis que l'autre moitié a trois temps (-> `3/4-mesure`), ce qui en fait une mesure mixte ! Il y a différentes façons de gérer cela dans osu! qui ont toutes leurs avantages et leurs inconvénients, que nous verrons dans le prochain segment.

Reviewer 2: ```suggestion
Maintenant pour vous donner un premier exemple de ceci dans osu!, regardez la difficulté `3/4` dans la beatmap fournie de la chanson "America" de Leonard Bernstein - Dans cette difficulté, nous avons tout chronométré en `3/4-signature`, ce qui signifie que chaque mesure est divisée en trois temps par des ticks blancs. Lorsque vous réglez le BeatSnapDivisor sur `1/2`, chaque temps est divisé en deux par les ticks rouges dans la timeline - Un tick rouge représentant une croche ♪. Cela fonctionnerait parfaitement bien pour une chanson en `3/4`, mais si vous écoutez attentivement, vous pouvez entendre quelque chose de spécial sur cette chanson : Alors que le tempo général reste (approximativement) le même, le regroupement rythmique des ♪ change presque à chaque mesure - La première mesure de la partie mappée à `00:03:662` est divisée en \[3+3\], car elle n'a qu'une seule accentuation de plus sur le tick rouge à `00:04:232`. La deuxième mesure, qui commence à `00:04:801` est regroupée en \[2+2+2\], car elle a deux accentuations supplémentaires sur les ticks blancs à `00:05:181` et `00:05:561`. En d'autres termes, toutes les mesures ont la même longueur (6♪s), mais la moitié d'entre elles n'ont que deux temps (-> `6/8-mesure`) tandis que l'autre moitié a trois temps (-> `3/4-mesure`), ce qui en fait une mesure mixte ! Il y a différentes façons de gérer cela dans osu! qui ont toutes leurs avantages et leurs inconvénients, que nous verrons dans le prochain segment.
```

Reviewer 2: pareil ici

![deux mesures de "America", la première est en 6/8, la seconde en 3/4](/wiki/shared/timing/I_like_to_be_in_america.png)

## Méthodes de timing des chansons avec des signatures en #/8

D'une manière générale, il y a 2 méthodes viables pour traiter ce problème dans osu!.

### Emulation des signatures #/8

> En parlant de la difficulté `3/4` de *America* de Leonard Bernstein, j'ai mentionné qu'un des temps se trouve sur un tick rouge au lieu d'un tick blanc à `00:04:232` parce que cette mesure est divisée en `\[3+3\], ce qui en fait une `6/8-mesure`. Afin de "fixer" l'emplacement du temps, le BPM doit être multiplié par `0,666`. De plus, la signature temporelle affichée dans le panneau TimingSetupPanel doit être réglée sur 2/4` - Ce n'est actuellement pas possible dans l'éditeur, mais il est possible de la changer dans le fichier `.osu` de la beatmap. Après avoir fait cela, le BeatSnapDivisor doit être changé en `1/3` de sorte qu'il divise un temps en trois ♪-notes, qui sont alors représentées par les ticks violets. Le résultat de cette opération est visible dans la difficulté 6/8.

Reviewer: ```suggestion
En parlant de la difficulté `3/4` de *America* de Leonard Bernstein, j'ai mentionné qu'un des temps se trouve sur un tick rouge au lieu d'un tick blanc à `00:04:232` parce que cette mesure est divisée en `\[3+3\], ce qui en fait une `6/8-mesure`. Afin de "fixer" l'emplacement du temps, le BPM doit être multiplié par `0,666`. De plus, la signature temporelle affichée dans le panneau TimingSetupPanel doit être réglée sur 2/4` - Ce n'est actuellement pas possible dans l'éditeur, mais il est possible de la changer dans le fichier `.osu` de la beatmap. Après avoir fait cela, le BeatSnapDivisor doit être changé en `1/3` de sorte qu'il divise un temps en trois notes ♪, qui sont alors représentées par les ticks violets. Le résultat de cette opération est visible dans la difficulté 6/8.
```

![150 BPM avec un métronome 2/4 en comparaison avec 100 BPM avec un métronome 3/4](/wiki/shared/timing/150bpm24vs100bpm34.png)

> Pour un exemple plus simple, regardez le jeu fourni de *We Are The Champions* de Queen, qui est une chanson purement en `6/8`. Lorsque vous regardez l'onglet timing de la difficulté `6/8`, vous pouvez voir que les petits ticks blancs entre les mesures correspondent exactement aux temps réels de la chanson, et que toutes les croches ♪ sont mises en correspondance en utilisant le `1/3-BeatSnapDivisor`.

Reviewer: ```suggestion
Pour un exemple plus simple, regardez le beatmapset fourni de la chanson *We Are The Champions* de Queen, qui est une chanson purement en `6/8`. Lorsque vous regardez l'onglet timing de la difficulté `6/8`, vous pouvez voir que les petits ticks blancs entre les mesures correspondent exactement aux temps réels de la chanson, et que toutes les croches ♪ sont mises en correspondance en utilisant le `1/3-BeatSnapDivisor`.
```

> Pour un exemple plus simple, regardez le beatmapset fourni de la chanson *We Are The Champions* de Queen, qui est une chanson purement en `6/8`. Lorsque vous regardez l'onglet timing de la difficulté `6/8`, vous pouvez voir que les petits ticks blancs entre les mesures correspondent exactement aux temps réels de la chanson, et que toutes les croches ♪ sont mises en correspondance en utilisant le `1/3-BeatSnapDivisor`.

Reviewer 2: ```suggestion
Pour un exemple plus simple, regardez le beatmapset fourni de la chanson *We Are The Champions* de Queen, qui est une chanson purement en `6/8`. Lorsque vous regardez l'onglet timing de la difficulté `6/8`, vous pouvez voir que les petits ticks blancs entre les mesures correspondent exactement aux temps réels de la chanson, et que tous les huitièmes de note ♪ sont mises en correspondance en utilisant le `1/3-BeatSnapDivisor`.
```

> Cette méthode est rythmiquement la plus précise dans osu!. Puisque les ticks blancs représentent toujours un temps, le logo osu!pulsant dans le SongSelectMenu et les effets sonores Nightcore sont toujours au point et transmettent correctement le "feeling" du morceau. Il est très facile et pratique d'utiliser cette méthode pour les chansons en signatures telles que `6/8` mais aussi `9/8`, `12/8`, etc. - Beaucoup de ces signatures sont plus courantes qu'on ne le pense, par exemple [Nekomata Master - Avalon no Oka](https://osu.ppy.sh/beatmapsets/317538) est un morceau en signature `12/8`, qui a été intuitivement chronométré en signature `4/4` et mappé en utilisant le `1/3-BeatSnapDivisor`. C'est pourquoi nous appliquons généralement cette méthode de chronométrage pour toutes les chansons en `#/8-signature` où les temps sont systématiquement divisés en trois croches ♪♪♪ (par exemple, elle serait appliquée sur We Are The Champions de Queen, alors qu'elle ne le serait pas sur America de Leonard Bernstein parce qu'elle est en mesure mixte).

Reviewer: ```suggestion
Cette méthode est rythmiquement la plus précise dans osu!. Puisque les ticks blancs représentent toujours un temps, le logo osu! pulsant sur l'écran de sélection de beatmap et les effets sonores du mod Nightcore sont toujours au point et transmettent correctement le "feeling" du morceau. Il est très facile et pratique d'utiliser cette méthode pour les musiques en signatures telles que `6/8` mais aussi `9/8`, `12/8`, etc. - Beaucoup de ces signatures sont plus courantes qu'on ne le pense, par exemple [Nekomata Master - Avalon no Oka](https://osu.ppy.sh/beatmapsets/317538) est un morceau en signature `12/8`, qui a été intuitivement chronométré en signature `4/4` et mappé en utilisant le `1/3-BeatSnapDivisor`. C'est pourquoi nous appliquons généralement cette méthode de chronométrage pour toutes les chansons en `#/8-signature` où les temps sont systématiquement divisés en trois croches ♪♪♪ (par exemple, elle serait appliquée sur We Are The Champions de Queen, alors qu'elle ne le serait pas sur America de Leonard Bernstein parce qu'elle est en mesure mixte).
```

Mais cette méthode peut aussi poser quelques problèmes : Tout d'abord, le BeatSnapDivisor n'a pas autant de couches de "triple-snaps" (3,6,12) que de "duple-snaps" (2,4,8,16). Cela peut rendre impossible l'enregistrement de rythmes rapides potentiels qui peuvent se produire dans certaines chansons, et que vous pourriez vouloir mapper dans des difficultés plus élevées.

> Deuxièmement, il existe des `#/8-signatures` plus complexes que celles qui ont des multiples de 3 dans le chiffre supérieur de la signature. Par exemple, alors qu'une mesure 6/8 est toujours formée de \[3+3\], vous pouvez obtenir des mesures avec des temps "irréguliers" en combinant des \[2\] et des \[3\]. Par exemple, une mesure "7/8" peut être formée comme suit : \[3+2+2\], \[2+2+3\] ou \[2+3+2\]. Si vous voulez toujours que les temps soient chronométrés précisément avec des ticks blancs, vous devez utiliser plusieurs sections de chronométrage dans une mesure pour suivre à la fois les temps "rapides" (les \[2\]s) et les temps "lents" (les \[3\]s). Ceci peut être vu dans la difficulté `7/8` de "Le Mystere Des Voix Bulgares - Erghen Diado". Comme vous pouvez le voir, nous avons dû définir deux sections de timing par mesure (marquées par des signets), et à cause des différentes valeurs de BPM, nous avons également dû changer le multiplicateur SV de la moitié des sections. Se déplacer dans cette map est aussi assez fastidieux, car bien que toutes les notes mappés soient de même longueur, la moitié d'entre elles sont snappées comme des notes `1/2` et l'autre moitié comme des notes `1/3`. C'est la raison pour laquelle nous n'appliquons pas cette méthode pour les chansons avec des signatures temporelles complexes (ou lorsque les rythmes deviennent impossibles à snapper), et nous autorisons la méthode alternative utilisée dans la difficulté "3.5/4".

Reviewer: ```suggestion
Deuxièmement, il existe des `#/8-signatures` plus complexes que celles qui ont des multiples de 3 dans le chiffre supérieur de la signature. Par exemple, alors qu'une mesure 6/8 est toujours formée de \[3+3\], vous pouvez obtenir des mesures avec des temps "irréguliers" en combinant des \[2\] et des \[3\]. Par exemple, une mesure "7/8" peut être formée comme suit : \[3+2+2\], \[2+2+3\] ou \[2+3+2\]. Si vous voulez toujours que les temps soient chronométrés précisément avec des ticks blancs, vous devez utiliser plusieurs sections de chronométrage dans une mesure pour suivre à la fois les temps "rapides" (les \[2\]s) et les temps "lents" (les \[3\]s). Ceci peut être vu dans la difficulté `7/8` de "Le Mystere Des Voix Bulgares - Erghen Diado". Comme vous pouvez le voir, nous avons dû définir deux sections de timing par mesure (marquées par des signets), et à cause des différentes valeurs de BPM, nous avons également dû changer le multiplicateur de slider velocity de la moitié des sections. Se déplacer dans cette map est aussi assez fastidieux, car bien que toutes les notes mappés soient de même longueur, la moitié d'entre elles sont snappées comme des notes `1/2` et l'autre moitié comme des notes `1/3`. C'est la raison pour laquelle nous n'appliquons pas cette méthode pour les chansons avec des signatures temporelles complexes (ou lorsque les rythmes deviennent impossibles à snapper), et nous autorisons la méthode alternative utilisée dans la difficulté "3.5/4".
```

### Les traiter comme des signatures #/4

> Cette méthode de timing est généralement très simple : les ♪-notes sont toujours représentées par les ticks rouges (et blancs) du diviseur `1/2`. Cela signifie que même lorsque la chanson change de signature, le BPM reste le même et le BeatSnapDivisor n'a pas besoin d'être changé en `1/3` afin de mapper les ♪-notes. La navigation dans la timeline est facile, et le BPM qui est affiché dans SongSelect et sur le site web est généralement un indicateur précis de la vitesse de frappe habituelle de la beatmap ("Vous pouvez singletap / stream 200 BPM" se réfère toujours à singletap ticks rouges / streaming ticks bleus dans ce tempo). Cependant, pour les signatures temporelles complexes, cela ne suit pas toujours précisément le rythme de la chanson, et ne suit jamais précisément le rythme des signatures temporelles composées - Par exemple, en regardant la difficulté `3/4` de "We Are The Champions" de Queen, vous pouvez voir que les petits ticks blancs entre les mesures ne se posent pas sur les rythmes de la chanson.

Reviewer: ```suggestion
Cette méthode de timing est généralement très simple : les notes ♪ sont toujours représentées par les ticks rouges (et blancs) du diviseur `1/2`. Cela signifie que même lorsque la chanson change de signature, le BPM reste le même et le BeatSnapDivisor n'a pas besoin d'être changé en `1/3` afin de mapper les notes ♪. La navigation dans la timeline est facile, et le BPM qui est affiché dans l'écran de sélection de beatmap et sur le site web est généralement un indicateur précis de la vitesse habituelle de la beatmap ("Vous pouvez singletap / stream 200 BPM" se réfère toujours à singletap les ticks rouges / streaming ticks bleus dans ce tempo). Cependant, pour les signatures temporelles complexes, cela ne suit pas toujours précisément le rythme de la chanson, et ne suit jamais précisément le rythme des signatures temporelles composées - Par exemple, en regardant la difficulté `3/4` de "We Are The Champions" de Queen, vous pouvez voir que les petits ticks blancs entre les mesures ne se posent pas sur les temps de la chanson.
```

> C'est la méthode à privilégier pour le timing des musiques avec des signatures temporelles complexes, car elle réduit considérablement l'effort de timing et assure une manipulation facile de la timeline. Vous avez toujours le droit d'utiliser la première méthode, plus précise, mais vous n'êtes obligé de l'utiliser que lorsque la valeur de battement de la chanson est constamment de trois croches ♪♪♪ (par exemple, les deux chronométrages fournis pour Erghen Diado sont acceptables, car sa valeur de battement est incohérente \[2+2+3\]). Pour les mesures mixtes, le début de chaque mesure doit atterrir sur une grande coche blanche. Les `#/2`- ou `#/16-signatures` composées et complexes doivent être traitées de la même manière.

Reviewer: ```suggestion
C'est la méthode à privilégier pour le timing des musiques avec des signatures temporelles complexes, car elle réduit considérablement l'effort de timing et assure une manipulation facile de la timeline. Vous avez toujours le droit d'utiliser la première méthode, plus précise, mais vous n'êtes obligé de l'utiliser que lorsque la valeur de battement de la chanson est constamment de trois croches ♪♪♪ (par exemple, les deux chronométrages fournis pour Erghen Diado sont acceptables, car sa valeur de battement varie \[2+2+3\]). Pour les mesures mixtes, le début de chaque mesure doit atterrir sur un grand tick blanc. Les signatures `#/2` ou `#/16` composées et complexes doivent être traitées de la même manière.
```

> C'est la méthode à privilégier pour le timing des musiques avec des signatures temporelles complexes, car elle réduit considérablement l'effort de timing et assure une manipulation facile de la timeline. Vous avez toujours le droit d'utiliser la première méthode, plus précise, mais vous n'êtes obligé de l'utiliser que lorsque la valeur de battement de la chanson est constamment de trois croches ♪♪♪ (par exemple, les deux chronométrages fournis pour Erghen Diado sont acceptables, car sa valeur de battement varie \[2+2+3\]). Pour les mesures mixtes, le début de chaque mesure doit atterrir sur un grand tick blanc. Les signatures `#/2` ou `#/16` composées et complexes doivent être traitées de la même manière.

Reviewer 2: ```suggestion
C'est la méthode à privilégier pour le timing des musiques avec des signatures temporelles complexes, car elle réduit considérablement l'effort de timing et assure une manipulation facile de la timeline. Vous avez toujours le droit d'utiliser la première méthode, plus précise, mais vous n'êtes obligé de l'utiliser que lorsque la valeur de battement de la chanson est constamment de trois huitièmes de note ♪♪♪ (par exemple, les deux chronométrages fournis pour Erghen Diado sont acceptables, car sa valeur de battement varie \[2+2+3\]). Pour les mesures mixtes, le début de chaque mesure doit atterrir sur un grand tick blanc. Les signatures `#/2` ou `#/16` composées et complexes doivent être traitées de la même manière.
```
